### DISCUSSION TITLE
Failed to fetch the Collabora capabilities endpoint: cURL error 6

### DISCUSSION THREAD

**Discussion by Martinligabue** [nextcloud/all-in-one#4522] [Questions] [answered]
Even if this is similar to other discussions, like #4421 or #4430 I think it has enough differences to put in a different discussion.
The error I get is the following:
```
[richdocuments] Error: Failed to fetch the Collabora capabilities endpoint: cURL error 6: Could not resolve host: <hostname> (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://<hostname>/hosting/capabilities
	from ? by -- at 8 apr 2024, 17:02:32
```
Nextcloud container logs end with:
```
Collabora URL (used for Nextcloud to contact the Collabora server):
  <hostname>
Collabora public URL (used in the browser to open Collabora):
  <hostname>
Callback URL (used by Collabora to connect back to Nextcloud):
  autodetected (will use the same URL as your user for browsing Nextcloud)

```
Collabora works nicely in my instance. I saved its logs as the following file.
[collabora-logs.txt](https://github.com/nextcloud/all-in-one/files/14922308/collabora-logs.txt)

**Comment by szaimen** [comment, marked as answer]
> Collabora works nicely in my instance

In that case I think you can ignore the error
